jax/numpy/lax_numpy.py

```diff
@@ -2625,6 +2625,25 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None,
   return true_divide(dot(X, X_T.conj()), f).squeeze()
 
 
+@_wraps(onp.corrcoef)
+def corrcoef(x, y=None, rowvar=True, bias=None, ddof=None):
+  c = cov(x, y, rowvar)
+  if isscalar(c) or (len(c_shape) != 1 and len(c_shape) != 2):
```

Comment:
You don't define `c_shape`, but `len(shape(c)) == 0` would seem to cover it.

Note that JAX doesn't distinguish between 0D arrays and scalars.